test/tc_xml.rb

```diff
@@ -66,6 +66,23 @@ def batch_test(parser)
     end
     assert_equal(count, 2)
   end
+
+  def test_skip_bad_xml_record
+    @parsers.each do | parser |
+      puts "\nRunning skip_bad_xml_record with: #{parser}.\n"
+      batch_skip_bad_xml_record_test(parser)
+    end
+  end
+
+  def batch_skip_bad_xml_record_test(parser)
+    reader = MARC::XMLReader.new('test/bad_record.xml', :parser=>parser)
+    count = 0
+    for record in reader
+      count += 1
+      assert_instance_of(MARC::Record, record)
+    end
+    assert_equal(count, 1)
+  end
```

Comment:
I'm not certain I follow how this test is testing that a skip happens. Because the test complete, and with a count of one rather than 2?  Can you leave comment to that effect?

Reply:
Hmm, I guess maybe it's not obvious.  But the test fixture has only two records in it.  It's a variation on the other tests that use similar approach but do not explicitly state there are 30 records in the fixture.